src/CodeCompliance/Domain/AbstractCodeComplianceCheck.php

```diff
@@ -15,6 +15,11 @@
 
 abstract class AbstractCodeComplianceCheck implements CodeComplianceCheckInterface
 {
+    /**
+     * @var string
+     */
+    protected const DOCUMENTATION_BASE_URL = 'https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/';
```

Comment:
What do you think about moving it to some configuration file
such as  Upgrader\Configuration\ConfigurationProvider or a new configuration file on CodeCompliance level

Reply:
Yes, I wanted to store it somewhere in the configuration, but didn't find a place that is on my opinion suitable for that. But I saw before only infrastructure config providers. `Upgrader\Configuration\ConfigurationProvider` looks more suitable for that, I'll have a look